### DISCUSSION TITLE
Is it posible to get info from a older .map file to remake a map?

### DISCUSSION THREAD

**Discussion by christooss** [mapsforge/mapsforge#1487] [Q&A] [answered]
There is a Chinese bike GPS computer that uses .map file type created with mapforge. Is there any way to "reverse engineer" or at least get as much info from a .map file possible. said map and rebuild it with newer information? Here is a zip of a map in question. I've tried opening it in Cartograph Maps and it worked without problem but limited info is seen there.

[SI00002303103E628502301C.map.zip](https://github.com/user-attachments/files/15902850/SI00002303103E628502301C.map.zip)

Any help would be appreciated so I can see if this is even possible.

**Comment by devemux86** [comment, marked as answer]
You must rebuild the map file from an updated osm file.

**Reply by christooss** [reply]
Can you generate osm based on .map file?

**Reply by devemux86** [reply]
The process is osm to map.

**Reply by christooss** [reply]
Ok, I then understood what tool does. I'll look into issue more. If there is a viewer/tool that provides more info about .map file. Cause my main question is not about generating osm file but how to get as much info from a .map file possible.

Thanks for the info

**Reply by devemux86** [reply]
You can study the library and see how it reads the map files.

**Comment by christooss** [comment]
This was not answered at all. But alas. I'll have to get help from somewhere else I guess.

**Reply by devemux86** [reply]
_Question is already answered that only possible process is osm > map._

The answers depend on how clear the questions are.

About the map info, what info?
Someone has to invest time and study the reader to do custom things.
